libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -62,6 +64,8 @@ pub struct BiomeRestResourceManager {
     // This is only used if the biome-credentials feature is enabled
     #[allow(dead_code)]
     user_store: Arc<SplinterUserStore>,
+    #[allow(dead_code)]
```

Comment:
What is the need for this?

Reply:
For now, that field is only used when the biome-credentials feature is enabled. So you get a lint warning when that feature is not enabled.

Reply:
I added a comment on the dead_code warning. I thought it would be better not conditionally compile that because that field is one I think other features will use once they are implemented. And I did not want to imply that that field should only be used for credentials feature